### PR TITLE
DOC-918 Added two parameters

### PR DESCRIPTION
**Pull request by sfc-gh-mheavin**
BROWSER_RESPONSE_TIMEOUT and MAXHTTPRETRIES

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,6 +123,8 @@ The following table lists all valid connection properties:
 | USER                       | Yes      | If AUTHENTICATOR is set to `externalbrowser` or the URL for native SSO through Okta, set this to the login name for your identity provider (IdP).     |
 | WAREHOUSE                  | No       |                                                                               |
 | CONNECTION_TIMEOUT         | No       | Total timeout in seconds when connecting to Snowflake. The default is 120 seconds |
+| BROWSER_RESPONSE_TIMEOUT   | No       | Number to seconds to wait for authentication in an external browser (default: 120). |
```
Comment (sfc-gh-dyoshinaga) [inline README.md:126]: It might be worth moving this after `AUTHENTICATOR`, since this seems to apply to `AUTHENTICATOR=externalbrowser`.

**Review by sfc-gh-dyoshinaga — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -123,6 +123,8 @@ The following table lists all valid connection properties:
...
+| MAXHTTPRETRIES             | No       | Maximum number of times to retry a failed HTTP requests (default: 7). You can can set `MAXHTTPRETRIES=0` to remove the retry limit, but doing so runs the risk of the .NET driver infinitely retrying failed HTTP calls. |
```
Comment (sfc-gh-mbartolome) [inline README.md:127]: two typos:

- "a failed HTTP requests" - singular or plural?
- "You can can set..."

**Comment by sfc-gh-mheavin** [comment]
I have read the CLA Document and I hereby sign the CLA

File: README.md (hunk 2 above)
Comment (sfc-gh-mheavin) [reply README.md:127]: Fixed

File: README.md (hunk 1 above)
Comment (sfc-gh-mheavin) [reply README.md:126]: Done. (Though I originally thought it was more like timeout :)

**Pull request merged**